examples/hub_demo/hub_demo.ino

```diff
@@ -420,3 +419,9 @@ void printProgStr(const char* str)
   while ((c = pgm_read_byte(str++)))
     Serial.print(c);
 }
+
+/* Create a word from 2 bytes */
+uint16_t makeWord( uint8_t h, uint8_t l )
+{
+  return (h << 8) | l ;
+}
```

Comment:
Please remove this, as this will cause conflicts when using Arduino: https://github.com/arduino/Arduino/search?utf8=%E2%9C%93&q=makeWord.